packages/lisk-api-client/test/types/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+export interface FakeApiClient {
```

Comment:
Why do we need to use FakeApiClient?

Reply:
@SargeKhan Main reason for fake type is because many properties on APIClient class are not optional, but unneeded for testing purposes

Reply:
@SargeKhan We need it to allow for stubs on apiClient in api_resource tests. In each resource test will use real APIClient class as per @ishantiw suggestion.